src/pages/Account/TransactionRows/SwapTransactionRow.tsx

```diff
@@ -85,10 +85,11 @@ export function SwapTransactionRow({ transaction }: SwapTransactionRowProps) {
       </TransactionDetails>
 
       <TransactionDetails>
-        {confirmedTime ? (
+        {confirmedTime ?? addedTime ? (
           <Flex flexDirection="column" fontSize="12px">
-            <Box>{DateTime.fromMillis(confirmedTime).toFormat('HH:mm:ss')}</Box>
-            <Box>{DateTime.fromMillis(confirmedTime).toFormat('dd/MM/yyyy')}</Box>
+            <Box>{DateTime.fromMillis(confirmedTime ?? addedTime).toFormat('HH:mm:ss')}</Box>
+            <Box>{DateTime.fromMillis(confirmedTime ?? addedTime).toFormat('dd/MM/yyyy')}</Box>
+            {!confirmedTime && <Box fontSize="8px">( ADDED TIME )</Box>}
```

Comment:
This looks a bit odd, why "added time" in uppercase?  Couldn't see how it looks on the preview if you have screenshot it would be great.

Also it should be a translated.

Reply:
<img width="970" alt="Screenshot 2022-09-14 at 20 40 06" src="https://user-images.githubusercontent.com/102727631/190236193-b32a7c6a-ddc9-4ea6-acb3-45fc905347c5.png">
It was upper case by css. So I added it as uppercase